source/_docs/getting-support.md

```diff
@@ -137,6 +137,8 @@ Access support history for chats in the User or Site Dashboards by selecting **S
 #### Can I start a chat session via email?
 No. Initiate chats via the Pantheon User or Site Dashboards.
 
+#### I can't access the Pantheon Dashboard login page, can I contact support via email?

```

Comment:
A form would be great - make it clear that is isn't to start a support ticket - other than to let us know they cannot access their dashboard.